packages/p2p-v2/src/pages/orders/screens/Orders/__tests__/Orders.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Orders from '../Orders';
+
+jest.mock('@deriv/api-v2', () => ({
+    p2p: {
+        order: {
+            useGetList: () => ({
+                data: [],
+                isFetching: false,
+                isLoading: false,
+                loadMoreOrders: () => undefined,
+            }),
+        },
+    },
+}));
+
+jest.mock('@/hooks', () => ({
+    useQueryString: () => ({ queryString: { get: () => 'Active orders' } }),
+}));
+
+jest.mock('../OrdersTable', () => ({
+    OrdersTable: () => <div data-testid='dt_p2p_v2_orders_table' />,
+}));
+
+jest.mock('../OrdersTableHeader', () => ({
+    OrdersTableHeader: () => <div data-testid='dt_p2p_v2_orders_table_header' />,
```

Comment:
do we need to use test id's here, can we just mock these components like this:

```suggestion
    OrdersTable: () => <div>OrdersTable</div>,
}));

jest.mock('../OrdersTableHeader', () => ({
    OrdersTableHeader: () => <div>OrdersTableHeader</div>,
```

Reply:
i dont think for the test case here it doesn't matter.. since it creates a div for both the cases..

Reply:
still tho, we should use getByText rather than getByTestId, since we should only get the test id as a last resort if we can't do anything else